File: itc.java
import java.util.Scanner;

public class BeverageSuggestion {
    public static void main(String[] args) {
        // Static variable to determine output.
        int varHot = 70;

        // Create a Scanner object for user input.
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the temperature outside today (in °F): ");

        // Input validation
        if (scanner.hasNextInt()) {
            int temp = scanner.nextInt();

            // The logic behind the madness.
            if (temp >= varHot) {
                System.out.println("Iced Tea");
            } else {
                System.out.println("Coffee");
            }
        } else {
            System.out.println("Please enter a valid number.");
        }

        // Close the scanner.
        scanner.close();
    }
}

File: itc.py
#Made by yours truly, use with caution.
#Adam Pacek II
#Static variables to determine output.
varCold = 69
varHot = 70

#Prompt for user input.
temp = input("Temperature outside today:")

#Convert user input into a readable int or integer.
temp = int(temp)

#The logic behind the madness.
if temp >= varHot:
    print("Iced Tea")
elif temp <= varCold:
    print("Coffee")